### ISSUE TITLE
image to image？

### ISSUE THREAD

**Issue by ucaswindlike** [YuvalNirkin/fsgan#77]
Great work!
I want to figure out how to translate a single image to another single image.

**Comment by YuvalNirkin** [comment]
[swap.py](https://github.com/YuvalNirkin/fsgan/blob/master/inference/swap.py) supports single images as well.

**Comment by gungorbasa** [comment]
@YuvalNirkin Yes, it support but for some reason I had to update `swap.py` to be able to achieve this. For some reason, it never entered `face_swapping(source[-1], target[-1], output, select_source, select_target)` part.

I know its not great but, I did something like below:
```
imageExtensions = (".jpg" or ".jpeg")
if imageExtensions in source[-1] and imageExtensions in target[-1] :
    face_swapping(source[-1], target[-1], output, select_source, select_target)
else:
    batch(source, target, output, face_swapping, postfix='.mp4', skip_existing=True)
```
